src/Quartz/Impl/AdoJobStore/Common/ConfigurationBasedDbMetadataFactory.cs

```diff
@@ -38,7 +40,11 @@ public ConfigurationBasedDbMetadataFactory(string sectionName, string providerNa
         /// <returns>The properties parser</returns>
         protected virtual PropertiesParser GetPropertiesParser()
         {
-            NameValueCollection settings = (NameValueCollection)ConfigurationManager.GetSection(sectionName) ?? new NameValueCollection();
+            NameValueCollection settings =
+#if CONFIGURATION
+                (NameValueCollection)ConfigurationManager.GetSection(sectionName) ??
```

Comment:
No, unfortunately not. The rationale is that different app models will have different config models that make sense for them (web apps use one model, store apps use another, etc.). So, we've implemented new config models in Framework layers that are targeted at specific app models (ASP.NET Core 1 has a new config infrastructure, for example), but there's nothing common for libraries to use. The guidance for libraries is to allow apps to pass configuration data in programatically and make the apps responsible for reading the actual config files.

In Quartz.Net's case, the quartz.config files still work since we read them directly, but there's no support for reading from an app.config like there was before.